src/Stan/Cli.hs

```diff
@@ -164,23 +166,52 @@ toggleSolutionP = flag ShowSolution HideSolution $ mconcat
     , help "Hide verbose solution information for observations"
     ]
 
+data ConfigCommand
+    = CheckCommand Check
+    | RemoveCommand Scope
+    | ObservationCommand (Id Observation)
```

Comment:
It's a pity that we can't reuse `partitionEithers` here easily :disappointed: Reminds of counting systems in old times:

* 1 constructor
* 2 constructors
* Many constructors

Reply:
Exactly :smile_cat: